parlai/mturk/core/react_server/dev/components/core_components.jsx

```diff
@@ -707,6 +707,37 @@ class ReviewButtons extends React.Component {
   }
 }
 
+class NextButton extends React.Component {
+  // This component is responsible for initiating the click
+  // on the next button to get the next subtask from the app
+
+  componentDidUpdate(prevProps, prevState, snapshot) {
+    if (this.props.onInputResize !== undefined) {
+      this.props.onInputResize();
+    }
+  }
+  render() {
+    let next_button = (
+      <button
+        id="next-button"
+        type="button"
+        className="btn btn-default btn-lg"
+        onClick={() => this.props.nextButtonCallback()}
+      >
+        <span
+          className="glyphicon glyphicon-chevron-right" aria-hidden="true" />{' '}
+        Next
```

Comment:
Small nit - maybe having the icon after the wording looks better here?

i.e. "Next >" instead of "> Next"